mantle/platform/qemu.go

```diff
@@ -958,10 +958,13 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 	case "bios":
 		break
 	case "uefi":
-		builder.setupUefi(false)
+		if err := builder.setupUefi(false); err != nil {
+			panic(err)
```

Comment:
This function returns an error...I think we should propagate it rather than panic.

Reply:
Fixed.